src/main/java/seedu/address/model/transaction/TransactionLog.java

```diff
@@ -38,6 +38,9 @@ public List<Transaction> getTransactionLog() {
      */
     public double calculateNetTransacted() {
         double result = 0;
+        if (transactionList.isEmpty()) {
+            return result;
+        }
```

Comment:
Perhaps these lines could be deleted since result would be 0 if the transactionList is empty.

Reply:
Okay will do thank you :)